test/sampleconsistency/sampleconsistency_test.go

```diff
@@ -85,8 +88,20 @@ func checkContains(t *testing.T, rl []string, src string) {
 		}
 		ir++
 	}
-	if is < len(sl) && best < len(sl) && best != -1 {
-		t.Fatalf("README.md file is missing line %d ('%s') from file '%s'", best, sl[best], src)
+
+	actionMsg := "All files required for running Helloworld sample apps are checked " +
+		"against README.md, the content of source files should be identical with what's " +
+		"in README.md file, the list of the files to be verified is the same set of files " +
+		"used for running sample apps, they are configured in `/test/sampleapp/config.yaml`. " +
+		"If an exception is needed the file can be configured to be copied as a separate step " +
+		"in `PreCommand` such as: " +
+		"https://github.com/knative/docs/blob/65f7b402fee7f94dfbd9e4512ef3beed7b85de66/test/sampleapp/config.yaml#L4"
```

Comment:
Seems like there should be a comment referencing this message from the current config and/or file copying implementation to ensure changes are followed through to this message.

Reply:
Moved the message to where the config and copy was defined and reference it from unit test